script/script.js

```diff
@@ -1,41 +1,161 @@
-function getFile(event) {
```

Comment:
This file is a duplicate of script/lib/script.js

Reply:
Done